build-system/pr-check.js

```diff
@@ -212,28 +212,34 @@ const command = {
     let docFiles = files.filter(isDocFile);
     timedExecOrDie(`${gulp} check-links --files ${docFiles.join(',')}`);
   },
-  buildRuntime: function() {
+  runPreBuildChecks: function() {
     timedExecOrDie(`${gulp} clean`);
     timedExecOrDie(`${gulp} lint`);
+  },
+  buildRuntime: function() {
+    timedExecOrDie(`${gulp} clean`);
     timedExecOrDie(`${gulp} build`);
```

Comment:
theoretically, we don't need `gulp build`.  `gulp build --css-only` should be enough.
i know it might fail some tests right now, can we add a TODO to investigate later?